### PR TITLE
Refactor imports to avoid circular import errors

### PR DESCRIPTION
**Pull request by NeonDaniel**
Move imports into functions where applicable
Update imports to directly reference classes
Tested with https://github.com/NeonGeckoCom/neon-utils/pull/424

### CODE REVIEW

File: ovos_config/locale.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from dateutil.tz import gettz, tzlocal
-import ovos_config
+from ovos_config.config import Configuration
```
Comment (JarbasAl) [inline ovos_config/locale.py:2]: why this change? this usually solves circular imports and is cleaner than importing inside a function

i think in this package we need special core and perhaps should simply import ovos_utils as utils and reference it that way, it should avoid the circular imports

Comment (NeonDaniel) [reply ovos_config/locale.py:2]: My idea here is to eventually deprecate the top-level imports so we don't have to init all the submodules when importing one of them

Comment (NeonDaniel) [reply ovos_config/locale.py]: Reverted these changes after more consideration since import refactoring would be a separate refactor on its own with more compat. testing to be done

**Review by ChanceNCounter — APPROVED** [review]
Just some import reordering. Tests clean. Blocking downstream.

I suggest holding for a few minutes, in case somebody who hasn’t been on sabbatical has a last minute objection, but I can’t imagine what that might be. Which I guess is my point…
